metabase/metabase
# Update copy for native questions during column mapping

Closes https://github.com/metabase/metabase/issues/41647

### Description

Update copy, new copy makes more sense

### How to verify

- you need to add native question with text and/or number parameter to the dashboard
- add a number/text filter, edit dashboard, select filter options for mapping and check the message inside card 

### Demo


https://github.com/metabase/metabase/assets/125459446/a98d7cf2-4d83-4f65-9c23-0df94e3239b6


https://github.com/metabase/metabase/assets/125459446/34dc6b7d-71b9-42a0-9f18-4490166efbe7



### Checklist

~- [ ] Tests have been added/updated to cover changes in this PR~
